pinot-common/src/main/java/org/apache/pinot/common/audit/AuditConfigManager.java

```diff
@@ -121,40 +58,18 @@ private static <T> T mapPrefixedConfigToObject(Map<String, String> clusterConfig
     return AuditLogger.OBJECT_MAPPER.convertValue(subsetConfig.toMap(), configClass);
   }
 
-  /**
```

Comment:
was the removal of all the java docs for methods intentional?

Reply:
yeah. It was just adding a lot of noise for internal methods without adding much value IMO